database/raft/raft.go

```diff
@@ -650,17 +645,15 @@ func (sv *Service) join(addr, baseURL string) error {
 		return errors.Wrap(err)
 	}
 
-	if raft.IsEmptySnap(raftSnap) {
-		log.Printkv(ctx, "at", "joined", "appliedindex", 0, "msg", "(empty snap)")
-	} else {
+	if !raft.IsEmptySnap(raftSnap) {
 		err = sv.state.RestoreSnapshot(raftSnap.Data, raftSnap.Metadata.Index)
 		if err != nil {
 			return errors.Wrap(err)
 		}
 		sv.confState = raftSnap.Metadata.ConfState
 		sv.snapIndex = raftSnap.Metadata.Index
-		log.Printkv(ctx, "at", "joined", "appliedindex", raftSnap.Metadata.Index)
 	}
+	log.Printkv(ctx, "at", "joined", "appliedindex", raftSnap.Metadata.Index)
```

Comment:
Okay, then any opinion on which one we should keep (new member vs existing member)? I will prob remove the serveJoin message since it would happen on an arbitrary existing member. This one on the new member is prob more salient.